examples/complete/main.tf

```
@@ -135,6 +140,7 @@ module "eks_cluster" {
 }
 
 module "eks_node_group" {
+  count   = var.cluster_auto_mode_enabled ? 0 : 1
```

```suggestion
  enabled = local.auto_mode_enabled
```
Let's use `enabled` instead of adding count. 
This way, there is no need to modify outputs like this: `try(module.eks_node_group[0].eks_node_group_role_arn, null)`